lighthouse-core/runner.js

```diff
@@ -156,6 +156,12 @@ class Runner {
       // LHR has now been localized.
       const lhr = /** @type {LH.Result} */ (i18nLhr);
 
+      // Save lhr to ./latest-run, but only if -A is used.
+      if (settings.auditMode) {
+        const path = Runner._getDataSavePath(settings);
```

Comment:
I noticed an lhr file will linger if you did a `-G`  ... I nearly wrote some code to delete the latest run folder before running, but that seems dangerous.

Reply:
> I nearly wrote some code to delete the latest run folder before running, but that seems dangerous.

thank goodness you didn't, that would have been horrifying to me and the important examples I keep in that folder due to its gitignore properties 😅